### DISCUSSION TITLE
Can't use function from refs like in Demo

### DISCUSSION THREAD

**Discussion by SolimanGit** [MelihAltintas/vue3-openlayers#109] [Q&A]
Hi ! So very simple request, i did as in the Demo but i can't access function with refs... Console.log(osm) does an output with everything but osm.value = null. 
My goal is to retrieve the resolutions from the tileLayer to try to not have a blurry map anymore.
PS: If you have something so prevent a background blurry map like leaflet you will have my deepest thanks
` const osm = ref(null)
console.log(osm)
`
`
<ol-tile-layer ref="osm">
            <ol-source-osm />
        </ol-tile-layer>`

The output of console.log(osm):
![image](https://user-images.githubusercontent.com/43494167/194321481-b91e474c-10c1-46fb-b555-027989f346f7.png)

**Comment by modelair** [comment]
mb u try to log it before map initialization?
